tests/cli/test_flow_run.py

```diff
@@ -300,10 +295,24 @@ async def test_when_num_logs_smaller_than_page_size_then_no_pagination(
                 timestamp=datetime.now(tz=timezone.utc),
                 flow_run_id=flow_run.id,
             )
-            for i in range(self.PAGE_SIZE - 1)
+            for i in range(num_logs)
         ]
         await orion_client.create_logs(logs)
 
+        return flow_run
+
+    return create_flow_run
+
+
+class TestFlowRunLogs:
+    PAGE_SIZE = 200
```

Comment:
This could be LOGS_DEFAULT_PAGE_SIZE, MAX_PAGE_SIZE or DEFAULT_PAGE_SIZE but I don't feel strongly about it.

Reply:
I agree, your suggestion is much clearer